### DISCUSSION TITLE
Unzipping release using bsdtar from stream results in non-executable duckdb

### DISCUSSION THREAD

**Discussion by jaraco** [duckdb/duckdb#11059] [Ideas]
### What happens?

I've attempted to "install" duckdb by downloading it from the releases page using wget and then passing that stream directly to bsdtar to extract it (saving the hassle of saving a file, extracting that file, then deleting the file). When I do so, however, the resulting `duckdb` binary is not executable due to a [known limitation](https://unix.stackexchange.com/a/487379/275034).

It would be nice if duckdb could either package releases using tarballs (.tar.gz format) or by using [this workaround](https://github.com/libarchive/libarchive/issues/1106#issuecomment-446266093) when generating the zip files to ensure they can be extracted with permissions from the stream.

e.g. `bsdtar --format=zip --options zip:experimental -cf duckdb.zip duckdb`

### To Reproduce

```
apt install -y libarchive-tools wget
wget --output-document - --no-verbose https://github.com/duckdb/duckdb/releases/download/v0.10.0/duckdb_cli-linux-aarch64.zip | bsdtar -x -p
./duckdb  # Permission denied
```

### OS:

Ubuntu 22.04 aarch64

### DuckDB Version:

0.10.0

### DuckDB Client:

n/a

### Full Name:

Jason R. Coombs

### Affiliation:

personal

### Have you tried this on the latest [nightly build](https://duckdb.org/docs/installation/?version=main)?

I have tested with a release build (and could not test with a nightly build)

### Have you tried the steps to reproduce? Do they include all relevant data and configuration? Does the issue you report still appear there?

- [X] Yes, I have

**Comment by szarnyasg** [comment]
@jaraco – Thanks for reporting this and proposing a workaround. We'll discuss this but given that it's a feature request, I cannot guarantee that it will be implemented. Will keep this thread updated.
